### PR TITLE
For consistency, use lml to re-implement the option parsers.

### PR DESCRIPTION
**Pull request by chfw**
For the future maintainers and new comers, who will any way have to study [lml](http://lml.readthedocs.io/en/latest/),  applying the same plugin management system wherever possible will help simplify the internals of pyecharts.

### CODE REVIEW

File: pyecharts/echarts/option.py [hunk 1]
```diff
@@ -1174,12 +1172,7 @@ def zaxis3D(
     return _zaxis3D
 
 
-@collectfuncs
```
Comment (chfw) [inline pyecharts/echarts/option.py]: in the future, I hope these wrapper functions will disappear step by step like this.

File: pyecharts/echarts/tooltip.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # coding=utf-8
 from pyecharts.echarts.json_serializable import JsonSerializable
+from pyecharts.echarts.option_manager import Option
 
 
+@Option(option_for="tooltip")
```
Comment (chfw) [inline pyecharts/echarts/tooltip.py]: And those wrapper functions become a class.

**Comment by chenjiandongx** [comment]
@chfw 这个 PR 我持不同意见，觉得你把事情变得复杂了

**Comment by chfw** [comment]
OK.

**Pull request closed without merging**